readme.md

```diff
@@ -20,6 +20,10 @@ See [Wiki](https://github.com/sbt/jacoco4sbt/wiki) for more details.
 
 ## Change Log
 
+* *2.3.0*
```

Comment:
Can you add " (unreleased)" here so that we don't confuse people who try to use it and can't find it in between when this is merged and a new version is released